iothub/device/src/HsmAuthentication/ModuleAuthenticationWithHsm.cs

```diff
@@ -35,7 +35,9 @@ internal ModuleAuthenticationWithHsm(ISignatureProvider signatureProvider, strin
             string moduleId,
             string generationId,
             TimeSpan sasTokenTimeToLive,
-            int sasTokenRenewalBuffer) : base(deviceId, moduleId, (int)sasTokenTimeToLive.TotalSeconds, sasTokenRenewalBuffer)
+            int sasTokenRenewalBuffer,
+            bool disposalBySdk)
+            : base(deviceId, moduleId, (int)sasTokenTimeToLive.TotalSeconds, sasTokenRenewalBuffer, disposalBySdk)
```

Comment:
This is an internal class, so I directly added the new param to the existing constructor.